helm/chaos-mesh/values.yaml

```diff
@@ -83,12 +83,12 @@ chaosDaemon:
 
   tolerations: []
 
-dashboard:
+chaosServer:
   create: false
 
-  serviceAccount: chaos-dashboard
+  serviceAccount: chaos-controller-manager
```

Comment:
We use the same SA as controller-manager?

Reply:
Most parts are the same, we use the same one at present and I will pull the other pr to add a new serviceAccount for Chaos Server.